Cargo.toml

```diff
@@ -105,3 +110,8 @@ assets = [
 
 [patch.crates-io]
 servo-freetype-sys = { path = "servo-freetype-proxy" }
+
+[patch.crates-io.winapi]
+git = "https://github.com/davidhewitt/winapi-rs.git"
+branch = "conpty"
+features = ["winuser", "synchapi", "roerrorapi", "winerror", "wincon"]
```

Comment:
Why is it necessary to use a patched winapi here? Any changes should probably be upstreamed. It would be impossible for us to maintain this out of tree.

Reply:
Yes, I'm working to upstream those changes at https://github.com/retep998/winapi-rs/pull/699. We can definitely wait for that to merge first. I'm hoping it'll be soon.

It adds the definition of the `*PseudoConsole` apis, though we only use the accompanying `HPCON` type here as we're now loading the apis dynamically. Could in theory define `HPCON` ourselves here but I think it's cleaner to use it from winapi (once merged).